cluster/gce/gci/configure-helper.sh

```diff
@@ -469,6 +473,9 @@ function start-kubelet {
   if [[ "${ALLOCATE_NODE_CIDRS:-}" == "true" ]]; then
      flags+=" --configure-cbr0=${ALLOCATE_NODE_CIDRS}"
   fi
+  if [[ -n "${FEATURE_GATES:-}" ]]; then
+     flags+=" --feature-gates=${feature_gates}"
```

Comment:
girish can you fix in your PR?

On Mon, Aug 22, 2016 at 3:33 PM, Girish Kalele <notifications@github.com>
wrote:

> In cluster/gce/gci/configure-helper.sh
> <https://github.com/kubernetes/kubernetes/pull/30468#discussion_r75771623>
> :
>
> > @@ -469,6 +473,9 @@ function start-kubelet {
> >    if [[ "${ALLOCATE_NODE_CIDRS:-}" == "true" ]]; then
> >       flags+=" --configure-cbr0=${ALLOCATE_NODE_CIDRS}"
> >    fi
> > +  if [[ -n "${FEATURE_GATES:-}" ]]; then
> > +     flags+=" --feature-gates=${feature_gates}"
>
> s/feature_gates/FEATURE_GATES/
>
> Fails kube-up for me.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/30468/files/51198f59da4cab0c2a17bafd09667e2ef7690fbd#r75771623>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AFVgVPwZYI1TVrKqIo-fhidol9KTPxOEks5qiiPUgaJpZM4JieFK>
> .
>